xo.config.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+const globals = require("globals");
+const { defineConfig } = require("eslint/config");
+const compatPlugin = require("eslint-plugin-compat");
+
+module.exports = defineConfig([
+  {
+    extends: [compatPlugin.configs["flat/recommended"]],
+    languageOptions: {
+      sourceType: "commonjs",
```

Comment:
I'm not 100% about this change. I'd expect script to still work, but apparently something has changed in ESLint v9 or xo and we get plenty of warnings about `strict` and `no-implicit-globals` without this.